testassets/InteropTestsWebsite/InteropTestsWebsite.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(LatestFramework)'!='true'">net6.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(LatestFramework)'=='true'">net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(LatestFramework)'!='true'">net7.0</TargetFrameworks>
```

Comment:
Not directly related to this change, but why are two different conditions needed here?

Reply:
Interop tests are also run from grpc/grpc repo, and may specify a different framework.